test/ffmpeg-qsv/vpp/denoise.py

```diff
@@ -39,8 +39,9 @@ def check_metrics(self):
     def compare(k, ref, actual):
       assert ref is not None, "Invalid reference value"
       assert abs(ref[-3] - actual[-3]) < 0.2, "Luma (Y) out of baseline range"
-      assert abs(ref[-2] - actual[-2]) < 0.2, "Cb (U) out of baseline range"
-      assert abs(ref[-1] - actual[-1]) < 0.2, "Cr (V) out of baseline range"
+      if self.caps.get("chroma", True):
+        assert abs(ref[-2] - actual[-2]) < 0.2, "Cb (U) out of baseline range"
+        assert abs(ref[-1] - actual[-1]) < 0.2, "Cr (V) out of baseline range"
 
```

Comment:
If chroma is False, then we should also check that it did not change.  For example:
```python

if self.caps.get("chroma", True):
  ...
else:
  assert actual[-2] == 100, "Cb(U) changed, but caps don't support DENOISE chroma"
  assert actual[-1] == 100, "Cr(V) changed, but caps don't support DENOISE chroma"
```
This will let us catch if driver adds support for it in the future or introduces a bug that changes chroma unintentionally.